scripts/memory_logger.py

```diff
@@ -599,16 +587,17 @@ def _getMemory(self):
     return 0
 
   def _getStack(self):
-    if self._darwin() == True:
-      stack_trace = LLDB()
-    else:
-      stack_trace = GDB()
-    tmp_pids = self._getPIDs()
-    if tmp_pids != {}:
-      last_pid = sorted([x for x in tmp_pids.keys()])[-1]
-      return stack_trace.getStackTrace(str(last_pid))
+    # Attach to and create a process object we can reuse later
+    if self.process is None:
+      tmp_pids = self._getPIDs()
+      if tmp_pids != {}:
+        last_pid = sorted([x for x in tmp_pids.keys()])[-1]
```

Comment:
I am removing the whole thing in my next merge request. I don't really need the last key... any one of them will do so:

`one_pid = tmp_pids.keys()[0]`